types/twine-sugarcube/macro.d.ts

```diff
@@ -162,9 +197,7 @@ export interface MacroAPI {
      *    }
      * });
      */
-    add(name: string | string[],
-        definition: MacroDefinition,
-        deep?: boolean): void;
+    add(name: string | string[], definition: MacroDefinition, deep?: boolean): void;
```

Comment:
This is mostly Ezsh's baby so I'm not going to request a change here.  If they want to, that's their business.

That said.  Unless it's the point of a PR or otherwise expected, it's generally considered poor practice to make only formatting changes to working code.

Reply:
Ah, I didn't notice that change. I ran the formatting tool (which seems to be advised by DefinitivelyTyped), which produced that.